actionview/lib/action_view/helpers/tags/base.rb

```diff
@@ -138,7 +138,7 @@ def sanitized_method_name
           end
 
           def sanitized_value(value)
-            value.to_s.gsub(/\s/, "_").gsub(/[^-\w]/, "").downcase
+            value.to_s.gsub(/\s/, "_").gsub(/[^-[[:word:]]]/, "").downcase
```

Comment:
Nice catch. `to_s` doesn't seem to be necessary, but I'll put it in just in case.